ARTICLE-3.markdown

```diff
@@ -38,23 +38,19 @@ The protocol for the use of ballots in any way may be determined by the Member B
 
 As provided in the Articles of Organization, the membership shall form a board to monitor the operations of the cooperative, meet regularly to co-create strategic plans, author proposals for member ratification, and manage regular affairs deemed more impactful than the Org Areas (see below) have authority to manage, but not yet so impactful as to require entire membership consensus. The Board shall work to include all substantially impacted members in all decision-making processes.
 
-The Board shall meet monthly and review reports from all Org Areas charged with managing the operations of the cooperative. All members shall be welcome to participate in all Board meetings (without voting participation). Non-member Collaborators may attend the Board meeting (without voting participation) if agreed upon by consensus of the Board. The Board has the authority to conduct meetings or parts of meeting that discuss information that is confidential to the Company, including but not limited to, legal, financial or other private matters, and may excuse from the meeting non-Investor Members as determined in the sole discretion of the Board.  
```

Comment:
Hmmm I can't attach this comment to Section 3.2, but there's some language around Invested and Working Members up there, after "Defer to the decision of the Board.

Reply:
Fixed it.